modules/google-analytics/wp-google-analytics.php

```diff
@@ -131,17 +134,20 @@ public function insert_code( $output = true ) {
 			$custom_vars[] = "_gaq.push(['_trackPageview']);";
 		}
 
+		$custom_vars = apply_filters( 'jetpack_wga_classic_custom_vars', $custom_vars );
```

Comment:
Can you please document this filter so that it can be parsed for developer.jetpack.com? Thanks!

Reply:
Fixed in https://github.com/Automattic/jetpack/pull/7817/commits/236f58bb17e289eaeb28be9296b7661cbae2abeb